app/upgrades/v0.0.12.go

```diff
@@ -0,0 +1,130 @@
+	storetypes "cosmossdk.io/store/types"
+	upgradetypes "cosmossdk.io/x/upgrade/types"
+	cosmosTypes "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/pokt-network/poktroll/app/keepers"
+	servicetypes "github.com/pokt-network/poktroll/x/service/types"
+	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
+	suppliertypes "github.com/pokt-network/poktroll/x/supplier/types"
+)
+
+// Upgrade_0_0_12 is the upgrade handler for v0.0.12 upgrade.
+//   - Before: v0.0.11
+//   - After: v0.0.12
+
+// This upgrade introduces a type change to RevSharePercent from float32 to uint64, which is introduced as a separate
+// protobuf field. As a result, we expect existing on-chain data to switch to default value.
+// Investigate the impact of this change on existing on-chain data.
+var Upgrade_0_0_12 = Upgrade{
+	PlanName: "v0.0.12",
+	CreateUpgradeHandler: func(mm *module.Manager,
+		keepers *keepers.Keepers,
+		configurator module.Configurator,
+	) upgradetypes.UpgradeHandler {
+		// Adds new parameters using ignite's config.yml as a reference. Assuming we don't need any other parameters.
+		applyNewParameters := func(ctx context.Context) (err error) {
+			logger := cosmosTypes.UnwrapSDKContext(ctx).Logger()
+			logger.Info("Starting parameter updates for v0.0.12")
+
+			// Add supplier module staking_fee per `config.yml`. The min stake is set to 1000000 upokt, but we avoid
+			// GetParams() to avoid potential protobuf issues and all networks have the same value (no need to read).
+			// Validate with: `poktrolld q supplier params --node=https://testnet-validated-validator-rpc.poktroll.com/`
+			supplierParams := suppliertypes.Params{
+				MinStake: &cosmosTypes.Coin{
+					Denom:  "upokt",
+					Amount: math.NewInt(1000000),
+				},
+				StakingFee: &cosmosTypes.Coin{
+					Denom: "upokt",
+					// TODO_IN_THIS_PR: 100upokt a good value?
+					Amount: math.NewInt(100),
+				},
+			}
+
+			// ALL parameters must be present when setting params.
+			err = keepers.SupplierKeeper.SetParams(ctx, supplierParams)
+			if err != nil {
+				logger.Error("Failed to set supplier params", "error", err)
+				return err
+			}
+			logger.Info("Successfully updated supplier params", "new_params", supplierParams)
+
+			// Add service module `target_num_relays` parameter per `config.yml`.
+			// We don't use `GetParams()` to avoid potential protobuf issues and all networks have the same value (no need to read).
+			serviceParams := servicetypes.Params{
+				AddServiceFee: &cosmosTypes.Coin{
+					Denom:  "upokt",
+					Amount: math.NewInt(1000000000),
+				},
+				TargetNumRelays: 100,
+			}
+			err = keepers.ServiceKeeper.SetParams(ctx, serviceParams)
+			if err != nil {
+				logger.Error("Failed to set service params", "error", err)
+				return err
+			}
+			logger.Info("Successfully updated service params", "new_params", serviceParams)
+
+			// Add tokenomics module `global_inflation_per_claim` parameter per `config.yml`.
```

Comment:
When we say `per `config.yml`.`, it confuses me a bit.

To we mean "in line with the hard-coded config.yml in the repo"?

If so, let's update the comments to avoid confusion.